src/library/scala/collection/View.scala

```diff
@@ -163,6 +163,64 @@ object View extends IterableFactory[View] {
     def iterator = partition.underlying.iterator.filter(x => partition.p(x) == cond)
   }
 
+  /** A view that splits an underlying collection into two views */
```

Comment:
This comment appears to be based on the one for `Partition` but both are inaccurate. These classes are not views.